packages/letters-of-credit/src/components/Pages/AlicePage/AlicePage.js

```diff
@@ -96,24 +97,23 @@ class AlicePage extends Component {
 			return (
     		<div id="alicePageContainer" className="alicePageContainer">
     		  <div id="aliceHeaderDiv" className="flexDiv aliceHeaderDiv">
-    		    <span className="aliceUsername" onClick={() => {this.handleOnClick('matias')}}> {username} </span>
+    		    <span className="aliceUsername" onClick={() => {this.handleOnClick('matias')}}>
```

Comment:
@jackson-ross can you take the links out please? "Alice, Customer of Bank of Argentina" should still be there but without a link, and we should just get rid of "Go to Bob"